src/bundle/Resources/views/admin/role/add.html.twig

```diff
@@ -1,39 +1,39 @@
-{% extends "@EzPlatformAdminUi/layout.html.twig" %}
+{% extends 'EzPlatformAdminUiBundle:admin/role:base.html.twig' %}
 
 {% trans_default_domain 'role' %}
 
-{% block breadcrumbs %}
+{% block breadcrumbsAdmin %}
```

Comment:
I know this is all over the place but we should use underscore for block names as this is more common notation.